dissect/target/plugins/os/unix/history.py

```diff
@@ -48,41 +53,80 @@ def commandhistory(self):
                 if ih in user_details.user.home:
                     continue
 
-            for file_ in user_details.home_path.iterdir():
-                if file_.name not in COMMAND_HISTORY_FILES:
+            for file in user_details.home_path.iterdir():
+                if file.name not in COMMAND_HISTORY_FILES:
                     continue
 
-                try:
-                    next_cmd_ts = None
-
-                    for line in file_.open("rt", errors="replace"):  # Ignore Non-UTF-8 characters in bash_history
-                        cmd_ts = None
-                        line = line.strip()
-
-                        # Skip empty lines
-                        if not line:
-                            continue
-
-                        if line.startswith("#"):  # Parse timestamp if possible
-                            matches = re.search(r"^#([0-9]{10})$", line)
-                            if matches:
-                                ts = matches.group(1)
-                                try:
-                                    next_cmd_ts = from_unix(float(ts))
-                                except (ValueError, TypeError):
-                                    continue
-                            continue
-
-                        if next_cmd_ts:
-                            cmd_ts = next_cmd_ts
-                            next_cmd_ts = None
-
-                        yield CommandHistoryRecord(
-                            ts=cmd_ts,
-                            command=line,
-                            source=str(file_),
-                            _target=self.target,
-                            _user=user_details.user,
-                        )
-                except Exception:
-                    self.target.log.exception("Failed to parse command history: %s", file_)
+                # TODO: Starting with Python 3.10 we can use pattern matching (PEP 634)
```

Comment:
We generally tend not to leave `TODO`'s in the code, we either do them or create a ticket for them. In this case I think it can be replaaced with `NOTE`.

```suggestion
                # NOTE: Starting with Python 3.10 we can use pattern matching (PEP 634)
```